client/state/sites/blog-stickers/reducer.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { merge } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { SITES_BLOG_STICKER_LIST_RECEIVE } from 'state/action-types';
+import { combineReducers, createReducer } from 'state/utils';
+
+export const items = createReducer(
+	{},
+	{
+		[ SITES_BLOG_STICKER_LIST_RECEIVE ]: ( state, action ) => {
+			return merge( {}, state, {
+				[ action.payload.blogId ]: action.payload.stickers,
```

Comment:
@samouri I tried exactly that, and it didn't seem to give me the behaviour I was after:

```export const items = keyedReducer(
	'blogId',
	createReducer( null, {
		[ SITES_BLOG_STICKER_LIST_RECEIVE ]: ( state, action ) => action.payload,
	} ),
);```